utils/intellisensePrepublish.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+
+const generateTokensArray = require('./generateTokensArray');
+
+// if the file is being run directly
+const generateTokensForPrepublish = () => {
```

Comment:
this is a utility function wrapper around the `generateTokensArray` function above. This will generate the tokens and then create a file with the tokens in the intellisense project. This utility gets run during the intellisense pre-packaging step so that it will always have the most up to date version of the tokens

Reply:
Looks great! Although, I don't think we need this to be a function. We can just execute this script top to bottom and leverage top level await if needed.